content/en/api/integrations_webhook/code_snippets/result.custom_variables_get.sh

```diff
@@ -0,0 +1,12 @@
+# Example Response 1
+{
+    "name": "<CUSTOM_VARIABLE_NAME>",
+    "value": "1234",
+    "is_secret": false
+}
+# Example Response 2
+{
+    "name": "<CUSTOM_VARIABLE_NAME_2>",
+    "value": null,
```

Comment:
Is value returned null when it's a credential? I wonder if we should change it for `*****` to be consistent with our other endpoints 🤔

Reply:
Do you want this implemented?